json-definitions/iva/defects/get/index.json

```diff
@@ -1,67 +1,36 @@
 {
-  "title": "Section IVA",
+  "title": "DefectGETIVA",
   "type": "object",
   "additionalProperties": false,
-  "required": [
-    "sectionNumber",
-    "sectionDescription",
-    "vehicleTypes",
-    "requiredStandards"
+  "required" : [
+    "euVehicleCategory",
+    "basic",
+    "normal"
   ],
   "properties": {
-    "sectionNumber": {
-      "type": ["string"]
+    "euVehicleCategory": {
```

Comment:
There is a slight disparity between the example json provided on the spike, and the interface agreed. Example json references array of string, interface only string.